numpyro/distributions/continuous.py

```diff
@@ -448,6 +448,85 @@ def tree_flatten(self):
         return super(TransformedDistribution, self).tree_flatten()
 
 
+class Gompertz(Distribution):
+    """Gompertz Distribution sampling utilities.
+
+    All derivations can be found in this paper (Lenart, 2012):
+        https://doi.org/10.1080/03461238.2012.687697
+    with the exception that we call the parameters `concentration` and `rate`,
+    as opposed to their "location" and "scale" terms.
+
+    The CDF is
+
+    .. math::
+        F(x) = 1 - \exp \left\{ \frac{\text{rate}}{\text{concentration}} * \left [ \exp\{\text{concentration} \cdot x\} - 1 \right ] \right\}
```

Comment:
Hi @jackpot-nfer, the implementation looks correct to me, but it seems to me that the names of the parameters are a bit strange. E.g. by using `rate`, I expected that `F(x)` should depend on `x / rate`, rather than `concentration * x`. Could you use the [wikipedia](https://en.wikipedia.org/wiki/Gompertz_distribution) derivation instead? I believe that you just need to change:
+ concentration -> 1 / rate
+ rate_over_conc -> concentration
+ rate -> concentration * rate
across the implementation.

Reply:
@fehiepsi would the following work for you?

```latex
\begin{align}
\text{concentration} &\rightarrow \frac{1}{\text{rate}}\\
\text{rate} &\rightarrow \frac{\text{concentration}}{\text{rate}}
\end{align}

$$F(x) = 1 - \exp \left\{ -\text{concentration} * \left [ \exp \left\{\frac{x}{\text{rate}}\right\} - 1 \right ] \right\}$$
```

If we return precisely to the wikipedia definition for [Gompertz](https://en.wikipedia.org/wiki/Gompertz_distribution) I think we might violate the analogy with the definition of the `Gamma` distribution in terms of concentration and rate parameters (as opposed to shape and scale) that [you mentioned above](https://github.com/pyro-ppl/numpyro/pull/1127#discussion_r692634197).

Alternatively, we could note that the current parameterization is quite similar to the form appearing in the [Gompertz-Makeham law of mortality](https://en.wikipedia.org/wiki/Gompertz%E2%80%93Makeham_law_of_mortality) where $\alpha \rightarrow \text{rate}$ and $\beta \rightarrow \text{concentration}$. Of course the Makeham term is absent here.

Reply:
> $$F(x) = 1 - \exp \left\{ -\text{concentration} * \left [ \exp \left\{\frac{x}{\text{rate}}\right\} - 1 \right ] \right\}$$

Yep, I vote for this. This matches wikipedia definition of shape and scale. In NumPyro Gamma.concentration = Wikipedia's Gamma.shape, and Gamma.rate = Wikipedia's Gamma.rate=1/Gamma.scale) - so we are using the same pattern here for Gompertz. In Gompertz-Makeham law of mortality, I saw `x` go along with `beta`, so beta should be the scale parameter (using `concentration` in place of `scale:=1/rate` is strange to me).

Reply:
I don't know why the "The Gompertz distribution and Maximum Likelihood Estimation of its parameters" paper uses the name "shape" for the scale parameter - this is not conventional I believe.